src/main/LibsodiumEncryptor.ts

```diff
@@ -98,8 +93,16 @@ export class LibsodiumEncryptor implements Encryptor {
     return key
   }
 
-  private toBufferFromJSON(jsonBuffer: JSONBuffer): Uint8Array {
-    return Buffer.from(jsonBuffer)
+  private convertBinaryFieldsToBuffers(obj: any): any {
```

Comment:
One of the rare times when it seems to make sense to use `any` since we have to do a recursive traversal.